src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -217,13 +217,21 @@ export const TokenRowWallets = component$<TokenRowWalletsProps>(
         .value;
       latestTokenPrice.value = (await data.next()).value["price"];
 
+      let previousTime = "1970-01-01T00:00:00Z";
```

Comment:
Could we immidiately do
`let previousTime = new Date("1970-01-01T00:00:00Z").getTime();`...

Reply:
done also for
const timeOfLatestBalanceChange = new Date(timestamp).getTime();